assignment3/WordFinder.java

```diff
@@ -0,0 +1,91 @@
+package com.katermar.assignment3;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+
+/**
+ * Created by katermar on 6/8/2017.
+ */
+
+/*
+class WordFinder to find words in a char grid
+ */
+public class WordFinder {
+    private Dictionary dictionary;
+
+    public WordFinder(Dictionary dictionary) {
+        this.dictionary = dictionary;
+    }
+
+    public Dictionary getDictionary() {

```

Comment:
Not needed. It's good to keep interface of the class as small as possible and remove all not needed code. Less things to worry about when maintaining code in the future.